### ISSUE TITLE
Cannot find unrealcv.ini

### ISSUE THREAD

**Issue by PierreBoyeau** [unrealcv/unrealcv#83]
Hi,

First of all thanks for the amazing work. I'm trying to fiddle with this [example](http://docs.unrealcv.org/en/master/tutorials/generate_images_tutorial.html) but I face the following issue.
In my `Engine/Binaries/Win64`UE folder, I cannot find any `unrealcv.ini` config file. I suppose it's normal, since I haven't created it yet. However, I don't know how to write it, and I can't find any examples of the repo. Would you have any samples? For instance, I would like to increase the size of my screenshots.
Thanks in advance!

**Comment by qiuwch** [comment]
Hi, You can get the ini file location using 'vget /unrealcv/status` this command will report where the ini file is.

**Comment by mikeroberts3000** [comment]
Hi @PierreBoyeau, this is what my unrealcv.ini looks like.

```
[UnrealCV.Core]
Port=9000
Width=512
Height=512
```

I have successfully changed the Width and Height parameters to change the resolution of the UnrealCV screenshots.

**Comment by PierreBoyeau** [comment]
Hi guys, first of all thanks to both of you for your help.
My issue is that when I type `vget /unrealcv/status` I only get the following output (I work on a personnal project) : 
```
ls Listening
Client Connected
9000
```
I tried to create a custom `unrealcv.ini` in my `Engine/Binaries/Win64` engine folder, but this doesn't seem to have any effect. 
Do you have an idea why this does not work? I have version 0.3.0.

**Comment by abhay-venkatesh** [comment]
I have this same issue. 

Can you tell us how this was resolved?

**Comment by qiuwch** [comment]
Hi, @abhay-venkatesh 

Which unrealcv version are you using?

**Comment by abhay-venkatesh** [comment]
Hi @qiuwch 

I am using v0.3.0

**Comment by qiuwch** [comment]
Config file is not implemented in v0.3.0. Do you know how to upgrade to a newer version?

**Comment by abhay-venkatesh** [comment]
@qiuwch understood. I will compile the latest version and use that. Thank you!
